v-next/hardhat-viem/.eslintignore

```diff
@@ -0,0 +1 @@
+/test/fixture-projects
```

Comment:
I'm not sure if this is the best way to ignore the `fixture-projects`. Let me know if you have a better approach.